src/providers/vod/vod-provider.ts

```diff
@@ -26,29 +28,37 @@ export class VodProvider extends Provider {
 
     let requests: Array<ProviderRequest> = [];
     if (thumbSubTypesFilter) {
-      requests.push({loader: ThumbLoader, params: {entryId: this._player.getMediaInfo().entryId, subTypesFilter: thumbSubTypesFilter}});
+      requests.push({loader: ThumbLoader, params: {entryId: this._player.config.sources.id, subTypesFilter: thumbSubTypesFilter}});
```

Comment:
we are actually planning to remove the sources form config one day so better not use sources too widely in our code.
You can use player.sources instead.